index.html

```diff
@@ -14,54 +14,61 @@
     <title>Portfolio</title>
   </head>
   <body>
-    <header>
+    <header class="header-desktop">
+      <nav>
       <h2 class="welcome"><a href="#">Welcome</a></h2>
-      <nav><i class="fa fa-bars" aria-hidden="true"></i></nav>
+        <i class="fa fa-bars" aria-hidden="true"></i>
+        <ul class="desktop-nav">
+          <li><a href="#portfolio-home">Portfolio</a></li>
+          <li><a href="#about-section">About</a></li>
+          <li><a href="#contact-section">Contact</a></li>
+        </ul>
+      </nav>
+      <section class="home-page">
+        <h2 class="intro">
+          Hey there. I'm Selma <br /><span
+            >I am a software 
+            developer</span>
+        </h2>
```

Comment:
- [ ] Hi you've done really well on the design of this section in the desktop version 👍. However, I've noticed that in your mobile version the `font-weight` is not correct. it should have a `font-weight` of `700`. the requirements need you to match this Figma template as closely as you can so you can practice styling to match a template. Please implement the correct `font-weight`s 🙏. 
## Your Desing (`font-weight: 600`)
![image](https://user-images.githubusercontent.com/18730106/144667520-625030b9-436f-48ae-b08b-f6ee4cff2c1b.png)

## Figma Design (`font-weight: 700`)
![image](https://user-images.githubusercontent.com/18730106/144667499-024e4174-f51c-4263-977f-d30e93791ead.png)